runbotics/runbotics-orchestrator-ui/src/src-app/store/slices/ProcessInstance/ProcessInstance.thunks.ts

```diff
@@ -8,7 +8,7 @@ import URLBuilder from '#src-app/utils/URLBuilder';
 import { ProcessInstanceRequestCriteria } from './ProcessInstance.state';
 
 const processInstancePageURL = (params: PageRequestParams<ProcessInstanceRequestCriteria>) => URLBuilder
-    .url('/api/process-instances-page')
+    .url('/api/process-instances/Page')
```

Comment:
As far as I know we were supposed to use uppercase letters only for actions (like e.g. UpdateAttribute) not for nouns (like Page). So please change to /page or eventually /GetPage

Reply:
we have already accepted this convention for process resource migration, maybe we should discuss it one more time?